aten/src/ATen/native/quantized/cuda/make_per_channel_quantized_tensor.cu

```diff
@@ -0,0 +1,36 @@
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cuda/Loops.cuh>
+
+namespace at {
+namespace native {
+
+Tensor make_per_channel_quantized_tensor_cuda(
+  const Tensor& self,
+  const Tensor& scales,
+  const Tensor& zero_points,
+  int64_t axis) {
+    std::vector<int64_t> expected_shape(rtensor.dim(), 1);
+    expected_shape[axis] = rtensor.size(axis);
+
+    Tensor dst = at::_empty_per_channel_affine_quantized(
+    self.sizes(),
+    scales,
+    zero_points,
+    axis,
+    self.options().dtype(toQIntType(self.scalar_type())));
+  AT_DISPATCH_QINT_TYPES(
+      dst.scalar_type(), "make_per_channel_quantized_tensor_cuda", [&]() {
+        auto iter = TensorIteratorConfig()
+          .check_all_same_dtype(false)
+          .add_output(dst)
+          .add_input(self)
+          .build();
+        gpu_kernel(iter, [] GPU_LAMBDA(underlying_t value) -> scalar_t {
+          return scalar_t(value);
```

Comment:
is this expected?

Reply:
That's how its done in the per_tensor one I was going off of

Reply:
It seems to be very similar to the per_tensor one. Any way we can de-duplicate the code? Not sure if we need a separate file for per-tensor vs per-channel for just this one function.

Reply:
I can put them in the same file and deduplicate the code